packages/emoji/emoji.1.0.0/descr

```diff
@@ -0,0 +1,13 @@
+Use emojis by name
+
+Names for byte code sequences of common emojis
+
+#require "emoji";;
```

Comment:
yea but why does that matter for the description? it comes up in opam info, or are you using it in web stuff as well?